src/main/java/io/vertx/core/net/impl/ConnectionBase.java

```diff
@@ -103,6 +107,13 @@ protected ConnectionBase(ContextInternal context, ChannelHandlerContext chctx) {
     closeFuture.onComplete(this::checkCloseHandler);
   }
 
+  public static boolean checkSendFileSupport(final EventLoop eventLoop) {
```

Comment:
we need to do that in a way that the IOUring class is not available, have a look at `Transport#nativeTransport` method

Reply:
correct, let me handle CCE

Reply:
already done